modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,189 @@
+
+```bash
+curl https://api.redpanda.com/v1beta2/operations/<operation-id>
+```
+
+The response contains the current state of the operation: `IN_PROGRESS`, `COMPLETED`, or `FAILED`.
+
+== Throughput tiers
+
+When you create a Dedicated or BYOC cluster, you select a throughput tier. Throughput tiers define the sizing of a cluster. Availability depends on the region and the cluster type.
+
+- xref:deploy:deployment-option/cloud/dedicated/dedicated-tiers.adoc[Dedicated Cloud throughput tiers and regions]
+- xref:deploy:deployment-option/cloud/byoc-tiers.adoc[BYOC throughput tiers and regions]
+
+== Create a cluster
+
+To create a new cluster, first create a resource group and network if you have not already done so.
+
+=== Create a resource group 
+
+Create a resource group by making a POST request to the xref:api:ROOT:cloud-api.adoc#post-/v1beta2/resource-groups[`/v1beta2/resource-groups`] endpoint. Pass a name for your resource group in the request body.
+
+[,bash]
+----
+curl -H 'Content-Type: application/json' \
+-H 'Authorization: Bearer <token>' \
+-d '{
+  "name": "<resource-group-name>"
+}' -X POST https://api.redpanda.com/v1beta2/resource-groups
+----
+
+A resource group ID is returned. Pass this ID later when you call the Create Cluster endpoint.
+
+=== Create a network
+
+Create a network by making a request to xref:api:ROOT:cloud-api.adoc#post-/v1beta2/networks[`POST /v1beta2/networks`].
+
+Choose a CIDR range that does not overlap with your existing VPCs or your Redpanda network.
+
+[,bash]
+----
+curl -d \
+'{
+  "cidr_block": "10.0.0.0/20",
+  "cloud_provider": "CLOUD_PROVIDER_GCP",
+  "cluster_type": "TYPE_DEDICATED",
+  "name": "<network-name>",
+  "resource_group_id": "<resource-group-id>",
+  "region": "us-west1"
+}' -X POST https://api.redpanda.com/v1beta2/networks 
+----
+
+This endpoint returns a <<long_running_operations,long-running operation>>. 
+
+=== Create a new cluster
+
+After the network is created, make a request to the xref:api:ROOT:cloud-api.adoc#post-/v1beta2/clusters[`POST /v1beta2/clusters`] with the resource group ID and network ID in the request body. 
+
+[,bash]
+----
+curl -d \
+'{
+  "cloud_provider": "CLOUD_PROVIDER_GCP",
+  "connection_type": "CONNECTION_TYPE_PUBLIC",
+  "name": "my-new-cluster",
+  "resource_group_id": "<resource-group-id>",
+  "network_id": "<network-id>",
+  "region": "us-west1",
+  "throughput_tier": "tier-1-gcp-um4g",
+  "type": "TYPE_DEDICATED",
+  "zones": [
+    "us-west1-a",
+    "us-west1-b",
+    "us-west1-c"
+    ]
+  }
+}' -X POST https://api.redpanda.com/v1beta2/clusters
+----
+
+The Create Cluster endpoint returns a <<long_running_operations,long-running operation>>. When the operation is completed, you can retrieve cluster details by calling xref:api:ROOT:cloud-api.adoc#get-/v1beta2/clusters/-id-[`GET /v1beta2/clusters/\{id}`], passing the cluster ID as a parameter.
```

Comment:
In our regular docs, we use property instead of parameter.

Reply:
@micheleRP Ah, I think both property and parameter can be used in the API context, but parameter is the more common term for the "specific value you use in the API URL path" https://swagger.io/resources/articles/best-practices-in-api-design/